examples/exclamation_topology/pyleus_topology.yaml

```diff
@@ -3,7 +3,8 @@
 # The topology used as example in the Storm tutorial has been implemented with
 # Pyleus. https://github.com/nathanmarz/storm/wiki/Tutorial
 name: exclamation_topology
-
+sleep_spout_wait_strategy_time_ms: 1000
```

Comment:
I'd like to keep this example as minimal as possible.
Instead of putting this here, would you mind adding a reference to this option in the documentation. This should go here http://yelp.github.io/pyleus/yaml.html